Source/ProgressionSystemRuntime/Public/LevelActors/PSStarActor.h

```diff
@@ -14,26 +14,50 @@ class PROGRESSIONSYSTEMRUNTIME_API APSStarActor : public AActor
 	// Sets default values for this actor's properties
 	APSStarActor();
 
+	/** Static helper function that plays any given star animation from various places. */
+	UFUNCTION(BlueprintCallable, Category= "C++")
+	bool TryPlayStarAnimation(UPARAM(ref) float& StartTimeRef, UCurveTable* AnimationCurveTable);
```

Comment:
`UCurbeTable*` fails to compile, please do `class UCurveTable*`

Reply:
achieved with https://github.com/h4rdmol/ProgressionSystem/pull/14/commits/e92e2cbda56e5eda90a1d789026c82e3b811965c